test/test.config.js

```diff
@@ -7,6 +7,6 @@ const path = require('path');
 config.mocha.tests.push(path.join(__dirname, 'mocha'));
 
 const cfg = config['vc-verifier'];
-cfg.ledgerHostname = 'genesis.veres.one.localhost:42443';
+cfg.ledgerHostname = 'ashburn.capybara.veres.one';
```

Comment:
The veres-one config landing specifically in this `vc-verifier` namespace violates the idea that we have `bedrock-did-io`.  `bedrock-did-io` should be the one place to configure communications with various DID ledgers.

We don't for instance want to have accidental drift between configuring `bedrock-did-io` to use the Veres One mainnet, and forgetting to configure `vc-verifier` which is still using testnet.